src/main/java/mServer/crawler/sender/arte/ArteCategoryFilmsDTO.java

```diff
@@ -4,22 +4,22 @@
 
 public class ArteCategoryFilmsDTO {
 
-    private final ArrayList<String> programIds = new ArrayList<>();
-    private int pages;
-    
-    public void addProgramId(String aProgramId) {
-        programIds.add(aProgramId);
-    }
-    
-    public ArrayList<String> getProgramIds() {
-        return programIds;
-    }
+  private final ArrayList<String> programIds = new ArrayList<>();
+  private boolean hasNextPage;
 
-    public int getPages() {
-        return pages;
-    }
-    
-    public void setPages(int aPages) {
-        pages = aPages;
-    }
+  public void addProgramId(String aProgramId) {
+    programIds.add(aProgramId);
+  }
+
+  public ArrayList<String> getProgramIds() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)